Assets/Python/GreatPeople.py

```diff
@@ -1185,51 +1204,61 @@ def setup():
 		iRenaissance,
 		"Charles Le Brun", # 17th
 		"Jean-Baptiste Lully", # 17th
-		"Jean-Antoine Watteau", # 17th
```

Comment:
Basically I replaced him with Élizabeth Vigée Le Brun who seems to me more noteworthy, and I felt he could be removed since the French GA list is already quite long. There isn't much more reason than that, let me know if you prefer him in